pnpm-workspace.yaml

```diff
@@ -0,0 +1,20 @@
+packages:
```

Comment:
_I can't add a comment to the `pnpm-lock.yaml` file, so I'll do it here._

The commercial and core repositories will have their own lock files. They are independent of each other, so they will also need to be updated individually whenever we change dependencies.